frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -550,6 +555,13 @@ public static Options getOptions() {
                             .argName("MODELS-STORE")
                             .desc("Model store location where models can be loaded.")
                             .build());
+            options.addOption(
+                    Option.builder("t")
```

Comment:
As we discussed, I don't think introduce a cli option is a must have.

Reply:
Done